setup.py

```diff
@@ -103,10 +103,11 @@ def txt_read(*paths):
 setup_args['install_requires'] =\
     [
         "numpy",
-        "matplotlib",
+        "matplotlib>=1.5",
+        "cartopy",
```

Comment:
It's part of [setuptools](https://pythonhosted.org/setuptools/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies). Not sure about `distutils`, but you're already using `setuptools` anyway.